packages/scheduler/type-definitions/index.d.ts

```diff
@@ -1,9 +1,21 @@
-import { Scheduler, Timeline, Timer } from '@most/types';
+import { Scheduler, Timeline, Timer, Time } from '@most/types';
 
-export function newClockTimer (): Timer;
-export function newTimeline (): Timeline;
+export type Now = () => Time;
+
+export type Clock = {
+  now: Now
+};
 
 export function newScheduler (timer: Timer, timeline: Timeline): Scheduler;
 export function newScheduler (timer: Timer): (timeline: Timeline) => Scheduler;
 
 export function newDefaultScheduler (): Scheduler;
+
+export function newClockTimer (clock: Clock): Timer;
+export function newTimeline (): Timeline;
+
+declare function newPlatformClock (): Clock;
```

Comment:
Lol! I copied from flow and then proceeded to delete the wrong keyword from each line 🤣